generator.rb

```diff
@@ -0,0 +1,59 @@
+def is_vowel?(letter)
+  case letter
+  when "a", "e", "i", "o", "u"
+    letter = true
+  else
+    letter = false
+  end
+  return letter
+end
+
+def run_generator
+  puts "Please enter your first word for the portmanteau. It must be at least 2 characters long."
+  word_A = gets.chomp
+  until word_A.length >= 2
+    puts "I'm sorry, this word is invalid. Please enter a word that is at least 2 characters long."
+    word_A = gets.chomp
```

Comment:
In Ruby, we typically name variables using all lowercases letters, so this would be `word_a`.